doc/wasmoo.rst

```diff
@@ -0,0 +1,104 @@
+.. _wasmoo:
+
+***************************************
+Wasm Compilation With Wasm_of_ocaml
+***************************************
+
+.. TODO(diataxis)
+
+   This is an how-to guide.
+
+Wasm_of_ocaml_ is a compiler from OCaml to WebAssembly (Wasm for
+short). The compiler works by translating OCaml bytecode to Wasm code.
+
+
+Compiling to Wasm
+=================
+
+Dune has full support for building wasm_of_ocaml libraries and executables transparently.
+There's no need to customise or enable anything to compile OCaml
```

Comment:
Should the documentation mention the external dependencies that are required for the build ?

Reply:
I now explicitly point to the github repository above.